src/App.jsx

```diff
@@ -14,45 +16,114 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+const SORT_TYPE_ALPHABET = 'Sort alphabetically';
+const SORT_TYPE_LENGTH = 'Sort by length';
+
+function getPreparedGoods(goods, { sortType, isReverse }) {
+  let preparedGoods = [...goods];
+
+  if (isReverse === 1) {
+    preparedGoods = preparedGoods
+      .sort((good1, good2) => preparedGoods.indexOf(good2)
+        - preparedGoods.indexOf(good1));
+  }
+
+  preparedGoods.sort((good1, good2) => {
```

Comment:
add the check, if we need to call the sort method

Reply:
<img width="619" alt="image" src="https://github.com/mate-academy/react_list-of-goods-js/assets/70220667/ff7394ed-0f49-4afe-a0a4-c874c775d2ac">